app/components/primer/layout_component.rb

```diff
@@ -11,6 +12,22 @@ class LayoutComponent < Primer::Component
     DEFAULT_SIDEBAR_COL = 3
     ALLOWED_SIDEBAR_COLS = (1..(MAX_COL - 1)).to_a.freeze
 
+    # @example 40|Default
+    #   <%= render(Primer::LayoutComponent.new) do |component| %>
+    #     <% component.with(:sidebar) { "Sidebar" } %>
+    #     <% component.with(:main) { "Main" } %>
+    #   <% end %>
+    #
+    # @example 40|Left sidebar
+    #   <%= render(Primer::LayoutComponent.new(side: :left)) do |component| %>
+    #     <% component.with(:sidebar) { "Sidebar" } %>
+    #     <% component.with(:main) { "Main" } %>
+    #   <% end %>
+    #
+    # @param responsive [Boolean] Whether to collapse layout to a single column at smaller widths.
+    # @param side [:symbol] Which side to display the sidebar on. <%= one_of(Primer::LayoutComponent::ALLOWED_SIDES) %>
```

Comment:
I don't know the conventions for YARD but should this be `Symbol` instead of `:symbol`?
```suggestion
    # @param side [Symbol] Which side to display the sidebar on. <%= one_of(Primer::LayoutComponent::ALLOWED_SIDES) %>
```